WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -78,7 +108,31 @@ private extension DashboardViewController {
     }
 
     func configureTitle() {
-        navigationItem.title = ServiceLocator.stores.sessionManager.defaultSite?.name ?? Localization.title
+        navigationItem.title = titleName
+    }
+
+    func configureSubtitle() {
+        if isDefaultSiteNameAvailable {
+            containerView.backgroundColor = .listForeground
+            storeNameLabel.text = ServiceLocator.stores.sessionManager.defaultSite?.name ?? Localization.title
```

Comment:
I've seen this pattern repeats quite a lot now! I think it would be good if we can extract it to its own variable.

Reply:
I've used the new variable `shouldShowStoreNameAsSubtitle `